files/es/web/accessibility/aria/index.md

```diff
@@ -0,0 +1,121 @@
+
+También es importante reconocer que algunas personas que confían en la tecnología de asistencia son reacias a actualizar su software por temor a perder la capacidad de interactuar con su computadora y navegador. Debido a esto, es importante [usar elementos HTML semánticos](/en-US/docs/Learn/Accessibility/HTML) siempre que sea posible, ya que el HTML semántico es mucho más compatible con la tecnología de asistencia.
+
+También es importante probar su ARIA creado con tecnología de asistencia real. Así como los emuladores y simuladores de navegador no son una solución efectiva para probar el soporte completo, las soluciones de tecnología de asistencia de emuladas no son suficientes para garantizar completamente la funcionalidad.
+
+## Tutoriales
+
+- [Introducción a ARIA](/en-US/docs/Web/Accessibility/An_overview_of_accessible_web_applications_and_widgets)
+  - : Una introducción rápida a cómo hacer que el contenido dinámico sea accesible con ARIA. Véase también el clásico [introducción a ARIA de Gez Lemon](https://dev.opera.com/articles/view/introduction-to-wai-aria/), de 2008.
+- [Videos de lectores de pantalla usando ARIA](https://zomigi.com/blog/videos-of-screen-readers-using-aria-updated/)
+  - : vea ejemplos reales y simplificados de toda la web, incluidos videos de ARIA "antes" y "después".
+- [Usando ARIA](https://w3c.github.io/using-aria/)
+  - : Una guía práctica para desarrolladores. Sugiere qué atributos ARIA usar en elementos HTML. Las sugerencias se basan en las realidades de la implementación.
+
+## Simple ARIA enhancements
+
+- [Mejora de la navegación de páginas con referencias de ARIA](https://www.paciellogroup.com/blog/2013/02/using-wai-aria-landmarks-2013/)
+  - : Una buena introducción al uso de los puntos de referencia de ARIA para mejorar la navegación de la página web para los usuarios de lectores de pantalla. [Consulte también las notas de implementación históricas de ARIA] (https://www.paciellogroup.com/blog/2011/07/html5-accessibility-chops-aria-landmark-support/) y ejemplos en sitios reales (actualizado en julio de 2011 ).
+- [Mejorando la accesibilidad de los formularios](/en-US/docs/Web/Accessibility/ARIA/forms)
+  - : ¡ARIA no es solo para contenido dinámico! Aprenda a mejorar la accesibilidad de los formularios HTML utilizando atributos ARIA adicionales.
+
+## ARIA para widgets con scripts
+
+- [Escribir widgets JavaScript navegables por teclado](/en-US/docs/Web/Accessibility/Keyboard-navigable_JavaScript_widgets)
+  -: Elementos integrados como {{HTMLElement("input")}}, {{HTMLElement("button")}} etc. tienen accesibilidad de teclado integrado. Si 'falsifica' estos con {{HTMLElement("div")}}s y ARIA, debe asegurarse de que sus widgets sean accesibles desde el teclado.
+- [Regiones en vivo](/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions)
```

Comment:
```suggestion
- [Regiones en vivo](/es/docs/Web/Accessibility/ARIA/ARIA_Live_Regions)
```

Reply:
@Vallejoanderson aquí veo un problema, el enlace sugerido aún no existe, observa el mensaje devuelto: 

Sorry, the page /es/docs/Web/Accessibility/ARIA/ARIA_Live_Regions could not be found.
![Captura de pantalla 2022-03-09 a las 17 26 45 p  m](https://user-images.githubusercontent.com/61764276/157530130-993105db-fdc1-49cf-ae27-88346d251d3e.png)